src/UI.ts

```diff
@@ -290,10 +290,6 @@ class UI {
       showBigPlayButton(modus)
     }
 
-    videoEl.addEventListener('waiting', () => {
-      showLoading(true)
-    })
-
```

Comment:
As discussed in Slack: I think this keeps the loading spinner from ever showing up. We need to check this again :)